static/js/components/AudioPlayer.js

```diff
@@ -0,0 +1,140 @@
+// @flow
+import _ from "lodash"
+import React, { useEffect, useRef, useCallback, useState } from "react"
+import { useSelector } from "react-redux"
+import Amplitude from "amplitudejs"
+
+import { currentlyPlayingAudioSelector } from "../lib/redux_selectors"
+
+export default function AudioPlayer() {
+  const seekBar = useRef()
+  const playPauseButton = useRef()
+  const currentlyPlayingAudio = useSelector(currentlyPlayingAudioSelector)
+  const [audioLoaded, setAudioLoaded] = useState(false)
+  const [audioPlaying, setAudioPlaying] = useState(false)
+
+  useEffect(
```

Comment:
small readability suggestion: it might be good to move the binding for the `amplitudeInitialized` up above this `useEffect` call

Reply:
same for the `playbackStarted` and `playbackPaused` callbacks